### PR TITLE
Shift the metrics collection rate from every 10 to 20 seconds

### PR DESCRIPTION
**Pull request by guscarreon**
Shift the metrics collection rate from one datapoint every 10 seconds to one every 20. Also update the graphana queries to keep the graphs reasonable

### CODE REVIEW

File: pbsmetrics/prometheus/prometheus.go [hunk 1]
```diff
@@ -56,7 +56,7 @@ const (
 func NewMetrics(cfg config.PrometheusMetrics) *Metrics {
 	// define the buckets for timers
 	timerBuckets := prometheus.LinearBuckets(0.05, 0.05, 20)
-	timerBuckets = append(timerBuckets, []float64{1.5, 2.0, 3.0, 5.0, 10.0, 50.0}...)
+	timerBuckets = append(timerBuckets, []float64{1.5, 2.0, 3.0, 5.0, 20.0, 50.0}...)
```
Comment (hhhjort) [inline pbsmetrics/prometheus/prometheus.go]: Why was this changed?

Comment (guscarreon) [reply pbsmetrics/prometheus/prometheus.go]: My bad, let me rollback this one.
